blog/tests/test_views.py

```diff
@@ -107,3 +107,28 @@ def test_post_edit(self):
                                     follow=True)
         self.assertContains(response, 'Test Text Edit')
         self.assertEqual(200, response.status_code)
+
+    def test_drafts(self):
```

Comment:
Please learn the existence of *negative tests*

Reply:
I added a few negative tests.
When I integrate the decorator then I will add the tests for the authentication